spec/requests/invites_controller_spec.rb

```diff
@@ -120,6 +120,44 @@
         expect(response.status).to eq(200)
       end
 
+      it "doesn't automatically redirect to the topic if the invite has expired" do
+        invite.update!(topics: [Fabricate(:topic)])
+        invite.update!(expires_at: 1.day.ago)
+
+        get "/invites/#{invite.invite_key}"
+        expect(response.status).to eq(200)
+
+        expect(response.body).to have_tag(:body, with: { class: "no-ember" })
+        expect(response.body).to include(I18n.t("invite.expired", base_url: Discourse.base_url))
+      end
+
+      it "shows the accept invite page when the user can access the topic, but the invite would also add them to a group" do
+        invite.update!(topics: [Fabricate(:topic)])
+        InvitedGroup.create!(invite: invite, group: group)
+
+        get "/invites/#{invite.invite_key}"
+        expect(response.status).to eq(200)
+        expect(response.body).to_not have_tag(:body, with: { class: "no-ember" })
```

Comment:
```suggestion
        expect(response.body).not_to have_tag(:body, with: { class: "no-ember" })
```

Today I learned that `not_to` and `to_not` are same thing. Ultra minor suggestion for consistency in a file :)